### PR TITLE
Spellerror

### PR DESCRIPTION
**Pull request by tim-zju**
correct spell errors

### CODE REVIEW

File: contributors/design-proposals/volume-ownership-management.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ package volume
 
 func ManageOwnership(path string, fsGroup int64) error {
     // 1. recursive chown of path
-    // 2. make path +setgid
+    // 2. make path + setgid
```
Comment (philips) [inline contributors/design-proposals/volume-ownership-management.md]: this isn't right either.

Comment (philips) [inline contributors/design-proposals/volume-ownership-management.md]: should be +setgid.

**Comment by pigmej** [comment]
Why are you creating `roll-back` commit? Why not to rewrite first commits correctly? (It's still in PR so no worries).

**Comment by k82cn** [comment]
@tim-zju , I usually use `git commit --amend` & `git push --force` to roll-back. You can try it :).

**Comment by philips** [comment]
The misspelled opinion is fixed in master and the other two fixes are incorrect so I am going to close this pull request.

Thank you for contributing to the project; please continue to identify and fix issues.

**Pull request closed without merging**